### PR TITLE
Update README.md

### LINKED ISSUE #4: Add readme.md file
add a readme file for this repo, explaining the mentioning the following conventions.

- 1 issue 1 pr policy.
- Label usage like documentation, beginner, intermediate, advanced, design, etc

### PR DESCRIPTION
**Pull request by adinalini**
Added contributing guidelines and resolved #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+Part of GirlScript's Uplift programme, this project aims at helping everyone looking for opportunities to leverage their 
+knowledge, skills, and potential to higher levels by working and collaborating on an open source project.
+
+### Some basic rules for contributing:
+
+### Pull Requests
+1. Remember, only one pull request per issue.
+2. Make sure you make the PR with proper details and description detailing what you have done.
+3. If not pushed, proper feecback will be given as to why. Act upon it and make the PR again.
```
[the remark this answers is not in the document]
Comment (adinalini) [reply README.md:11]: I wasn't changing the content of the PR, I forgot to assign it to issue #4. I couldn't find a way to change the description of the previous PR.

[the remark this answers is not in the document]
Comment (adinalini) [reply README.md:11]: Okay just found the update comment function, honestly didn't see it that time.  😅

[the remark this answers is not in the document]
Comment (YashKumarVerma) [reply README.md:11]: git commit --amend

[the remark this answers is not in the document]
Comment (adinalini) [reply README.md:11]: Got it.

**Comment by YashKumarVerma** [comment]
Use the request review option to get it merged when the changes are done.

**Comment by YashKumarVerma** [comment]
@adinalini updates ?

**Comment by adinalini** [comment]
I'm done with the template, but since it's gonna be a general readme explaining the project, I decided to wait till after tommorow's meeting when we decide on the project.

**Comment by YashKumarVerma** [comment]
We can change it later on. please send the PR.

**Comment by adinalini** [comment]
Closing this PR as I deleted the fork and reforked, will push a new PR

**Pull request closed without merging**